### PR TITLE
bug: LT-2264 HMRC Query Reference Name

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: applications/forms/hmrc.py [hunk 1]
```diff
@@ -30,3 +30,12 @@ def query_explanation_form(application_id):
         default_button_name=strings.Hmrc.QueryExplanation.BUTTON_TEXT,
         back_link=back_to_task_list(application_id),
     )
+
+
+def reference_name_form():
+    return Form(
+        title="Enter a reference for this HMRC Query",
```
Comment (Mark-Spiers-kainos) [inline applications/forms/hmrc.py]: lite content

Comment (BenAndrew-Kainos) [reply applications/forms/hmrc.py]: Yeah sorry just realised

File: applications/forms/hmrc.py [hunk 2]
```diff
@@ -30,3 +30,12 @@ def query_explanation_form(application_id):
...
         back_link=back_to_task_list(application_id),
     )
+
+
+def reference_name_form():
+    return Form(
+        title=applications.HMRCQuery.InitialQuestions.REFERENCE_NAME_TITLE,
+        description=applications.HMRCQuery.InitialQuestions.REFERENCE_NAME_DESCRIPTION,
+        questions=[TextInput(name="name"),],
+        default_button_name="Continue",
```
Comment (stephanieboyd) [inline applications/forms/hmrc.py]: Oh, shouldn't this be in strings?

Comment (BenAndrew-Kainos) [reply applications/forms/hmrc.py]: Damn it

Comment (BenAndrew-Kainos) [reply applications/forms/hmrc.py]: Done

**Pull request merged**